src/index.ts

```diff
@@ -6,3 +6,11 @@ export {
   toPascal,
   objectToPascal,
 } from './caseConvert';
+export type {
+	ObjectToCamel,
```

Comment:
not sure why there is extra space here.. There definitely is not. 
![image](https://user-images.githubusercontent.com/9469090/221013422-9ad0db88-0504-4146-8599-0e4918fa7d95.png)